sections/futures/TradeCrossMargin/MarginInfoBox.tsx

```diff
@@ -0,0 +1,230 @@
+		const remainingMargin = crossMarginFreeMargin.sub(marginDelta);
+		return remainingMargin.add(potentialTrade.data?.margin || zeroBN);
+	}, [crossMarginFreeMargin, marginDelta, potentialTrade.data?.margin]);
+
+	const getPotentialAvailableMargin = useCallback(
+		(previewTrade: FuturesPotentialTradeDetails | null, marketMaxLeverage: Wei | undefined) => {
+			let inaccessible;
+
+			inaccessible = previewTrade?.notionalValue.div(marketMaxLeverage).abs() ?? zeroBN;
+
+			// If the user has a position open, we'll enforce a min initial margin requirement.
+			if (inaccessible.gt(0)) {
+				if (inaccessible.lt(previewTrade?.minInitialMargin ?? zeroBN)) {
+					inaccessible = previewTrade?.minInitialMargin ?? zeroBN;
+				}
+			}
+
+			// check if available margin will be less than 0
+			return previewTotalMargin.sub(inaccessible).gt(0)
+				? previewTotalMargin.sub(inaccessible).abs()
+				: zeroBN;
+		},
+		[previewTotalMargin]
+	);
+
+	const previewAvailableMargin = React.useMemo(() => {
+		const potentialAvailableMargin = getPotentialAvailableMargin(
+			potentialTrade.data,
+			marketInfo?.maxLeverage
+		);
+		return potentialAvailableMargin;
+	}, [potentialTrade.data, marketInfo?.maxLeverage, getPotentialAvailableMargin]);
+
+	const potentialMarginUsage = useMemo(() => {
+		if (!potentialTrade.data) return zeroBN;
+		const notionalValue = potentialTrade.data.notionalValue.abs();
+		const maxSize = totalMargin.mul(potentialTrade.data.leverage);
+		return notionalValue.div(maxSize);
+	}, [potentialTrade.data, totalMargin]);
+
+	const previewTradeData = React.useMemo(() => {
+		const size = wei(nativeSize || zeroBN);
+
+		return {
+			showPreview: !size.eq(0) || !marginDelta.eq(0),
+			totalMargin: potentialTrade.data?.margin || zeroBN,
+			freeAccountMargin: crossMarginFreeMargin.sub(marginDelta),
+			availableMargin: previewAvailableMargin.gt(0) ? previewAvailableMargin : zeroBN,
+			size: potentialTrade.data?.size || zeroBN,
+			leverage: potentialTrade.data?.margin.gt(0)
+				? potentialTrade.data.notionalValue.div(potentialTrade.data.margin).abs()
+				: zeroBN,
+			marginUsage: potentialMarginUsage.gt(1) ? wei(1) : potentialMarginUsage,
+		};
+	}, [
+		nativeSize,
+		marginDelta,
+		potentialTrade.data?.margin,
+		previewAvailableMargin,
+		potentialTrade.data?.notionalValue,
+		potentialTrade.data?.size,
+		crossMarginFreeMargin,
+		potentialMarginUsage,
+	]);
+
+	const showPreview = previewTradeData.showPreview && !potentialTrade.data?.showStatus;
+
+	return (
+		<>
+			<StyledInfoBox
+				dataTestId="market-info-box"
+				details={{
+					'Free Account Margin': {
+						value: formatDollars(crossMarginFreeMargin),
+						valueNode: (
+							<PreviewArrow showPreview={showPreview}>
+								{potentialTrade.status === 'fetching' ? (
+									<MiniLoader />
+								) : (
+									formatDollars(previewTradeData.freeAccountMargin)
+								)}
+							</PreviewArrow>
+						),
+					},
+					'Market Margin': !editingLeverage
```

Comment:
I think we may still want to show this, or potentially show the difference in margin. A user who is adding margin to reduce their leverage can not see the specific amount they are depositing. Instead, they see how much their free margin is reduced and need to do the mental math:
<img width="391" alt="image" src="https://user-images.githubusercontent.com/10401554/188018096-946df449-3d74-4fc9-8766-ed0c011d98bd.png">

Reply:
I removed it based on feedback from one of the product elites, they seemed to think it wasn't that helpful or not common when working with traditional cross margin UI. If you feel strongly I'd be happy to add it back but I think it would need to be in place of maybe free account margin as this list is growing too big otherwise.

Reply:
No worries, this way works